src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java

```diff
@@ -21,13 +25,26 @@ public class WebServiceIntegrationTest {
             new ResourceConfigurationSourceProvider()
     );
 
-
     @Test
-    void getEmployees_shouldReturnListOfEmployees() {
+    void getEmployees_shouldReturnListOfRoles() {
         List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
                 .request()
                 .get(List.class);
         assertTrue(response.size() > 0);
     }
 
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles_withValidBandName() {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode response = APP.client().target("http://localhost:8080/api/job-roles")
+                .request()
+                .get(JsonNode.class);
+        List<JobRole> JobRoleList = mapper.convertValue(
+                response, new TypeReference<List<JobRole>>(){}
+        );
+
+        Assertions.assertEquals(JobRoleList.get(0).getBandName(), "Executive");
```

Comment:
can you check this as part of an existing test to reduce the number of http calls required

Reply:
done